blenderproc/command_line.py

```diff
@@ -128,20 +177,31 @@ def cli():
 
         # Run either in debug or in normal mode
         if args.mode == "debug":
-            p = subprocess.Popen([blender_run_path, "--python-use-system-env", "--python-exit-code", "0", "--python", os.path.join(repo_root_directory, "blenderproc/debug_startup.py"), "--", path_src_run if not is_config else args.file, temp_dir] + unknown_args, env=used_environment)
+            # pylint: disable=consider-using-with
+            p = subprocess.Popen([blender_run_path, "--python-use-system-env", "--python-exit-code", "0", "--python",
+                                  os.path.join(repo_root_directory, "blenderproc/debug_startup.py"), "--",
+                                  path_src_run if not is_config else args.file, temp_dir] + unknown_args,
+                                 env=used_environment)
+            # pylint: enable=consider-using-with
         else:
-            p = subprocess.Popen([blender_run_path, "--background", "--python-use-system-env", "--python-exit-code", "2", "--python", path_src_run, "--", args.file, temp_dir] + unknown_args, env=used_environment)
+            # pylint: disable=consider-using-with
+            p = subprocess.Popen([blender_run_path, "--background", "--python-use-system-env", "--python-exit-code",
+                                  "2", "--python", path_src_run, "--", args.file, temp_dir] + unknown_args,
+                                 env=used_environment)
+            # pylint: enable=consider-using-with
 
         def clean_temp_dir():
             # If temp dir should not be kept and temp dir still exists => remove it
             if not args.keep_temp_dir and os.path.exists(temp_dir):
                 print("Cleaning temporary directory")
                 shutil.rmtree(temp_dir)
 
-        # Listen for SIGTERM signal, so we can properly cleanup and and terminate the child process
+        # Listen for SIGTERM signal, so we can properly clean up and terminate the child process
+        # pylint: disable=unused-argument
         def handle_sigterm(signum, frame):
             clean_temp_dir()
             p.terminate()
+        # pylint: enable=unused-argument
```

Comment:
I think we can also fix that by prepending _
```suggestion
        def handle_sigterm(_signum, _frame):
            clean_temp_dir()
            p.terminate()
```

Reply:
Oh that's nice.